can_tools/scrapers/official/federal/HHS/facility.py

```diff
@@ -105,6 +105,21 @@ def normalize(self, data: str) -> pd.DataFrame:
         temp = temp + df.eval(f"{pps} / {ppc}").fillna(0.0)
         df.loc[:, "inpatient_beds_used_covid_7_day_canavg"] = temp.values
 
+        # Calculate adult and pediatric hospital admissions (same process as above).
+        adult_admissions = "previous_day_admission_adult_covid_confirmed_7_day_sum"
+        adult_coverage = "previous_day_admission_adult_covid_confirmed_7_day_coverage"
```

Comment:
Out of curiosity, do we know for sure if CDC is using "confirmed" or "suspected" data?

Reply:
It looks like they use confirmed data, as that's what's specified in the footnotes here: https://covid.cdc.gov/covid-data-tracker/#county-view